linode/networkingipassignment/framework_resource_schema.go

```diff
@@ -0,0 +1,29 @@
+package networkingipassignment
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var frameworkResourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"id": schema.StringAttribute{
+			Computed:    true,
+			Description: "The ID of the IP assignment operation.",
+		},
+		"region": schema.StringAttribute{
+			Required:    true,
+			Description: "The region for the IP assignments.",
+		},
+		"assignments": schema.ListAttribute{
+			ElementType: types.ObjectType{
+				AttrTypes: map[string]attr.Type{
+					"address":   types.StringType,
+					"linode_id": types.Int64Type,
+				},
+			},
+			Optional: true,
+		},
+	},
+}
```

Comment:
Same as https://github.com/linode/terraform-provider-linode/pull/1658/files#r1878492658

Reply:
Made similar changes to this resource too